frontend/models/TextClassification.js

```diff
@@ -26,11 +26,13 @@ class TextClassificationRecord extends BaseRecord {
     explanation,
     multi_label,
     predicted_as,
+    event_timestamp,
```

Comment:
This will happen for all tasks. Could you include this change at main `BaseRecord` level?

Reply:
sure!